packages/tasks/src/tasks/audio-to-audio/spec/output.json

```diff
@@ -0,0 +1,25 @@
+{
+	"$id": "/inference/schemas/audio-to-audio/output.json",
+	"$schema": "http://json-schema.org/draft-06/schema#",
+	"description": "Outputs of inference for the Audio To Audio task",
+	"title": "AudioToAudioOutput",
+	"type": "array",
+	"items": {
+		"description": "A generated audio file with its label.",
+		"type": "object",
+		"properties": {
+			"label": {
+				"type": "string",
+				"description": "The label of the audio file."
+			},
+			"content-type": {
```

Comment:
Maybe?
Can you link to where those parameters are documented please?

```suggestion
			"content_type": {
```

Reply:
It's not documented AFAIK. I only know it from the call made by the widget on https://huggingface.co/tasks/audio-to-audio :confused: 

![image](https://github.com/huggingface/huggingface.js/assets/11801849/62c518ff-bb2c-4063-bdf3-7a2ce040f897)

Reply:
For `content-type` vs `content_type` I've added a normalizer rule in Python to accept both since the Python attribute can only be `content_type` (and is generated like this).

Reply:
You can see some specification from https://github.com/huggingface/api-inference-community/blob/main/docker_images/speechbrain/app/pipelines/audio_to_audio.py and potential outputs